app/controllers/Books.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../../database/models')
+const responses = require('../helpers/responses')
+
+class Books {
+  static async GetAllBooks(req, res) {
+    try {
+      const results = db.Book.findAll();
```

Comment:
you need to `await` this call to the database.

Reply:
Worked on it.